assets/sass/colors/_theme.scss

```diff
@@ -10,3 +10,7 @@ $aplus-brand-secondary: #EF353B;
 // strongly not recommended at the moment.
 $aplus-background: #FFF;
 $aplus-foreground: #000;
+
+// Additional colors used in some components and/or pages
+$layout-base-color: #236aa7;
```

Comment:
I actually created the variable because that specific colour was used in a couple of places, and I did not want to have the same colour hardcoded in each property. I will explore the `choose-contrast-color` function and I will see what can be done with that function.

Reply:
I will remove the variable. Using the hexadecimal colour value directly in the CSS properties should work fine.

Reply:
OK, let's merge it now. I thought you might rename the variable. The concern with removing the variable is that now a certain shade of blue is hardcoded in some CSS hover effects, but if some other A+ site (namely Tampere) changes the primary color Sass variable, then will the hardcoded shade of blue work very well with other primary colors? If it doesn't, then we must either add new Sass variables or calculate helper colors with a function that uses the primary color variable as input.